Resources/Prototypes/Entities/Clothing/Shoes/boots.yml

```diff
@@ -91,6 +91,9 @@
     sprite: Clothing/Shoes/Boots/laceups.rsi
   - type: Clothing
     sprite: Clothing/Shoes/Boots/laceups.rsi
+  - type: Tag
+    tags:
+      - LaceUpShoes
```

Comment:
seems like you added tag here but forgot to add the bounty

Reply:
wrong

Reply:
where

Reply:
its there

Reply:
It was a bounty I added and removed because the item was too cheap